tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/ProgrammeMembershipServiceImpl.java

```diff
@@ -350,4 +389,39 @@ private void updatePersonWhenStatusIsStale(Long personId) {
       personRepository.save(person);
     }
   }
+
```

Comment:
@JosephKelly / @Judge40 - I wonder if I could get your opinion on this? The rest of the PR is very standard stuff, but there is the little quirk below in relation to saving a record into two tables. It could be that we can live with this, given the parallel-saves are intended to be temporary while we finish the ProgrammeMembership refactoring, but I'm wondering if there might be a more elegant way of dealing with the issue. Happy to chat about it anytime!